backend/src/main/kotlin/org/loculus/backend/service/files/FilesDatabaseService.kt

```diff
@@ -22,8 +24,26 @@ class FilesDatabaseService(private val dateProvider: DateProvider) {
         return id
     }
 
+    fun getGroupId(fileId: FileId): Int? = getGroupIds(setOf(fileId))[fileId]
+
     fun getGroupIds(fileIds: Set<FileId>): Map<FileId, Int> =
         FilesTable.select(FilesTable.idColumn, FilesTable.groupIdColumn)
             .where { FilesTable.idColumn inList fileIds }
             .associate { Pair(it[FilesTable.idColumn], it[FilesTable.groupIdColumn]) }
+
+    fun setPublicAtIfEmpty(fileIds: Set<FileId>) {
```

Comment:
I find `setPublicAtIfEmpty` _very_ hard to understand. How about "setPublishedAtNowIfNotPublished"? Or just "setPublishedAtNow" and leave the rest implied?

Reply:
`setPublishedAtNow` sounds a bit misleading to me: the name indicates to me that it would overwrite the field with the date for now. What about just `publish()`?

Reply:
publish is good for me!